tools/bin/commands/release

```diff
@@ -56,7 +56,7 @@ get_release_version() {
 
 release::run() {
     source "$(basedir)/commands/util/maven_funcs"
-    source "$(basedir)/commands/util/operator_funcs"
+    # source "$(basedir)/commands/util/operator_funcs"
```

Comment:
Perhaps we remove it rather than comment it out?

```suggestion
```

Reply:
This belongs to the `gren` bits, which I commented out and will add back once `release` works again